README.md

```diff
@@ -1 +1,13 @@
-# ISTEX_MentalRotation
\ No newline at end of file
+# ISTEX_MentalRotation

```

Comment:
Please leave this file empty to be filled later for the main repositpry readme. Instead, you should build the same fime but in a sub-directory for this LDA clustering